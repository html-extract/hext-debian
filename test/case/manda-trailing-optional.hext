<a class:a_class/>
<?div class:div_class/>
