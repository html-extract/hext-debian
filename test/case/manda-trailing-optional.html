<html>
<body>
  <a class="first-a"></a>
  <div class="optional-div"></div>
  <a class="second-a"></a>
</body>
</html>
