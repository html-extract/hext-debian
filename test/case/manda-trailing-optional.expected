{"a_class":"first-a","div_class":"optional-div"}
{"a_class":"second-a"}
